crates/apollo-compiler/CHANGELOG.md

```diff
@@ -21,7 +21,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Fixes
 - **Validate against reserved names starting with `__` in schemas - [SimonSapin], [pull/923].**
-- **Validate the default value of input fields and arguments - [SimonSapin], [pull/925].**
+- **Fix duplicate diagnostic for variable with invalid default value - [SimonSapin], [pull/925].**
```

Comment:
maybe add this PR to the changelog entry too? if someone looks at 925 alone they might get the wrong impression